### DISCUSSION TITLE
Acessar form values de uma requisição POST

### DISCUSSION THREAD

**Discussion by bpegorari** [HashLoad/horse#271] [Q&A] [answered]
Olá Pessoal! Alguém poderia me ajudar com um code snippet para acessar os valores contidos em form values de uma requisição POST?

**Comment by viniciussanchez** [comment]
Não apareceu no ContentFields?

**Comment by gabrielbaltazar** [comment, marked as answer]
Seria dessa forma aqui.

![image](https://user-images.githubusercontent.com/9014016/172007901-423b2e04-cdc9-4a97-9482-491d3f5d55a0.png)
